### ISSUE TITLE
The result on pascal_voc 2007 validation set is bad.

### ISSUE THREAD

**Issue by gauss-clb** [nilboy/tensorflow-yolo#32]
I  have tested your tiny-yolo on pascal_voc 2007 validation set. The result of test is so bad, I think.  Do you use 
paramters from darknet tiny_yolo or trained by yourself?  Can your tiny_yolo reach MAP 50%?

**Comment by gauss-clb** [comment]
![test](https://user-images.githubusercontent.com/11674304/32947035-ef80b8a2-cbd4-11e7-85d5-1528a0f68276.png)
![test](https://user-images.githubusercontent.com/11674304/32947183-9027b044-cbd5-11e7-8f81-093ffda0229f.png)

![test](https://user-images.githubusercontent.com/11674304/32947101-3e2ad582-cbd5-11e7-8f25-e2be41bfa473.png)

**Comment by ss199302** [comment]
@gauss-clb Have you solved the problem of tiny_yolo?

**Comment by gauss-clb** [comment]
I'm sorry, I think it is too difficult to reproduce the MAP of yolo. Maybe the author use many tricks. And I'm also wonder whether the difference of `tensorflow` and `darknet` cause the difference in MAP.

**Comment by ss199302** [comment]
thank you 。
